hummingbot/connector/exchange/latoken/latoken_utils.py

```diff
@@ -0,0 +1,50 @@
+from typing import Any, Dict
+
+from hummingbot.client.config.config_methods import using_exchange
+from hummingbot.client.config.config_var import ConfigVar
+
+CENTRALIZED = True
+EXAMPLE_PAIR = "LA-USDT"
+DEFAULT_FEES = [0.1, 0.1]
```

Comment:
we put default shema instead of list
```suggest
DEFAULT_FEES = TradeFeeSchema(maker_percent_fee_decimal=Decimal("0.001"), taker_percent_fee_decimal=Decimal("0.001"))
```